### ISSUE TITLE
inconsistent use of tabs and spaces in indentation

### ISSUE THREAD

**Issue by nobicycle** [openzim/gutenberg#80]
Thank you for gutenberg zim.

The requirements and virtualenv were setup ok, but:

```
(gut-env) [me@tower gutenberg]$ gut-env/bin/python -V
Python 3.6.8
(gut-env) [me@tower gutenberg]$ ./gutenberg2zim 
  File "./gutenberg2zim", line 170
    if LANGUAGES == []:
                      ^
TabError: inconsistent use of tabs and spaces in indentation
```

**Comment by kelson42** [comment]
@dattaz @rgaudin any idea how to fix that bug?

**Comment by dattaz** [comment]
Fix in #81
